app/index.html

```diff
@@ -587,6 +587,30 @@ <h3 style="text-align: center">Select a preset for the palette</h3>
         </div>
     </div>
 </div>
+<div id="confirm-logout-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog" style="width:26%; padding-top:250px;">
+        <div class="modal-content">
+            <div style="margin-top:30px; float:left; padding-left:50px;">
+                <img src="images/confirm.png">
+            </div>
+            <div>
+                <div style="padding:15px;">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                    <h4 style="text-align: center">Are you sure you want to exit?</h4>
+                </div>
+                <div style="padding:15px; text-align:center;">
```

Comment:
Use the class `text-center` instead of applying text-align:enter as an inline style